aries-site/src/data/structures/components.js

```diff
@@ -850,6 +850,7 @@ export const components = [
   },
   {
     name: 'Menu',
+    accessibility: 'Passed WCAG 2.2 AA',
```

Comment:
Since 2.4.7 is failing I don't think we are meeting AA

Reply:
I think this tag should say `Failed WCAG 2.2 AA`

Reply:
changed